policyengine_us/variables/gov/irs/tax/federal_income/alternative_minimum_tax/alternative_minimum_tax.py

```diff
@@ -0,0 +1,133 @@
+            0.5,
+            1.0,
+        )
+        tax_rate_threshold = amt.brackets.thresholds[-1] * bracket_fraction
+        lower_rate = amt.brackets.rates[0]
+        higher_rate = amt.brackets.rates[1]
+        lower_tax = min_(reduced_income, tax_rate_threshold) * lower_rate
+        higher_tax = max_(0, reduced_income - tax_rate_threshold) * higher_rate
+
+        # Line 7
+        reduced_income_tax = lower_tax + higher_tax
+        dwks10, dwks13, dwks14, dwks19, e24515 = [
+            add(tax_unit, period, [variable])
+            for variable in [
+                "dwks10",
+                "dwks13",
+                "dwks14",
+                "dwks19",
+                "unrecaptured_section_1250_gain",
+            ]
+        ]
+        form_6251_part_iii_required = np.any(
+            [
+                variable > 0
+                for variable in [
+                    dwks10,
+                    dwks13,
+                    dwks14,
+                    dwks19,
+                    e24515,
+                ]
+            ]
+        )
+
+        # Complete Form 6251, Part III
+
+        line37 = dwks13
+        line38 = e24515
+        line39 = min_(line37 + line38, dwks10)
+        line40 = min_(reduced_income, line39)
+        line41 = max_(0, reduced_income - line40)
+        line42 = amt.brackets.calc(line41)
+        line44 = dwks14
+        cg = amt.capital_gains.brackets
+        line45 = max_(0, cg.thresholds["1"][filing_status] - line44)
+        line46 = min_(reduced_income, line37)
+        line47 = min_(line45, line46)
+        cgtax1 = line47 * cg.rates["1"]
+        line48 = line46 - line47
+        line51 = dwks19
+        line52 = line45 + line51
+        line53 = max_(0, cg.thresholds["2"][filing_status] - line52)
+        line54 = min_(line48, line53)
+        cgtax2 = line54 * cg.rates["2"]
+        line56 = line47 + line54
+        line57 = where(line41 == line56, 0, line46 - line56)
+        linex2 = where(line41 == line56, 0, max_(0, line54 - line48))
+        cgtax3 = line57 * cg.rates["3"]
+        line61 = where(
+            line38 == 0,
+            0,
+            0.25
+            * max_(0, (reduced_income - line41 - line56 - line57 - linex2)),
+        )
+        line62 = line42 + cgtax1 + cgtax2 + cgtax3 + line61
+        line64 = min_(reduced_income_tax, line62)
+        line31 = where(form_6251_part_iii_required, line64, reduced_income_tax)
+
+        # Form 6251, Part II bottom
+        line32 = tax_unit("foreign_tax_credit", period)
+        line33 = line31 - line32
+        regular_tax_before_credits = tax_unit(
+            "regular_tax_before_credits", period
+        )
+        lump_sum_distributions = tax_unit(
+            "form_4972_lumpsum_distributions", period
+        )
+        tax_unit("capital_gains_tax", period)
+        tax_before_credits = (
+            regular_tax_before_credits + regular_tax_before_credits
+        )
```

Comment:
This is the piece that was added in this PR

Reply:
Can you explain why `regular_tax_before_credits` is being added to itself?
What am I missing?

Reply:
is this resolved?

Reply:
Yes